plugins/hls-module-name-plugin/hls-module-name-plugin.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      2.4
 name:               hls-module-name-plugin
-version:            1.0.0.0
+version:            1.0.0.1
```

Comment:
We haven't had `1.0.0.0` on Hackage, do we need this bump?

Reply:
Oh, really? I thought I released one... I will revert these.